sdks/java/core/src/test/java/org/apache/beam/sdk/io/WriteTest.java

```diff
@@ -297,7 +297,7 @@ public void populateDisplayData(DisplayData.Builder builder) {
   @Test
   public void testWriteUnbounded() {
     PCollection<String> unbounded = p.apply(CountingInput.unbounded())
-        .apply(ToString.of());
```

Comment:
Are the other violations going to be separate PRs or part of this PR. Should we discuss the method name here or on the list?